content/docs/command-reference/exp/branch.md

```diff
@@ -0,0 +1,74 @@
+# exp branch
+
+Apply the changes from a previously run experiment into the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc exp branch [-h] [-q | -v] experiment branch
+
+positional arguments:
+  experiment     Experiment to be promoted.
+  branch         Git branch name to use.
+```
+
+## Description
+
+This command will directly promote an experiment to a Git branch.
```

Comment:
Partially core question: What does "promote" mean? Could the command be called `experiments promote` instead of `branch`? Why "branch"? Thanks

Reply:
"promote" is a bit leftover from the old experiments revision, but the general idea is that experiment commits and refs are ephemeral and do not show up in the "standard" git repo. "promoting" an experiment refers to making an experiment ref into a "formal" git branch or commit (either by converting it to a branch directly using this command, or by using `exp apply` + `git add/commit` to commit it into an existing git branch). Again, not sure what the ideal terminology will be for us here

Reply:
I see. `branch` seems intuitive in that case. The explanation could be more explicit like "make a Git branch based on this experiment" but I still don't fully get it: does it also make a first commit to that branch? Or just a branch in the current commit but having applied the experiment changes to the working tree?

p.s. I think we can avoid the term "promote" everywhere since it's unclear what we mean.

Reply:
> either by converting it to a git branch directly using this command, or by using exp apply + git add/commit

That's a good explanation for the description BTW, something like "This is equivalent to `dvc exp apply <experiment>; git checkout -b <branch>; git commit -m "<something>"`" — or whatever the process is.